code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -92,4 +92,21 @@
 		to_chat(H, SPAN_NOTICE("There is nothing there. You feel safe."))
 	return TRUE
 
-
+/datum/ritual/cruciform/base/sense_cruciform
+	name = "Cruciform sense"
+	phrase = "Et si medio umbrae"
+	desc = "very short litany to identify cruciform followers. Targets individuals directly in front of caster or being grabbed by caster."
+	power = 20
+
+/datum/ritual/cruciform/base/sense_cruciform/perform(mob/living/carbon/human/H, obj/item/weapon/implant/core_implant/C)
+	var/mob/living/carbon/human/T = get_victim(H)
+	if(T)
+		var/obj/item/weapon/implant/core_implant/cruciform/CI = get_implant_from_victim(H, /obj/item/weapon/implant/core_implant/cruciform, FALSE)
+		if(CI)
+			to_chat(H, "<span class='rose'>[T] has a cruciform installed.</span>")
+		else
+			fail("There is no cruciform on [T]", H, C)
+	else
+		fail("No target. Make sure your target is either in front of you or being grabbed by you.", H, C)
+		return FALSE
+	return TRUE
```

Comment:
No newline here.

Reply:
Can't believe you got me to sit down and open up my editing software for a newline
![image](https://user-images.githubusercontent.com/24533979/91460841-efb58d00-e84d-11ea-95c2-e2da0e0c6110.png)

Reply:
Newlines ain't gonna write themselves... well they could...